public/pages/Indices/containers/IndicesActions/index.tsx

```diff
@@ -309,7 +310,7 @@ export default function IndicesActions(props: IndicesActionsProps) {
                     {
                       name: "Split",
                       "data-test-subj": "Split Action",
-                      disabled: !selectedItems.length || selectedItems.length > 1,
+                      disabled: !selectedItems.length || selectedItems.length > 1 || selectedItems[0].data_stream !== null,
```

Comment:
for data stream backing index, only the write one could not do split/shrink operation, other are allowed.

Reply:
You are right, there is no indicator to tell whether the stream index is the write index. One possible solution is to call data stream API to get the one with largest generation of that data stream. May implement the logic later since it's concern case that user want to split data stream index